model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -50,10 +50,26 @@ class InsertReverseChannels(BackReplacementPattern):
     """
     enabled = False
 
+    @staticmethod
+    def get_fw_index(node: Node, idx: int):
+        if node.has_valid('rt_info'):
```

Comment:
I think function with less levels of if looks better like 
if not node.has_valid('rt_info'):
    return idx
etc

Reply:
Done.